src/utils/helpers.ts

```diff
@@ -119,4 +125,21 @@ function buildEventOptions(eventName: string): GitFitCodeEventOptions {
   ];
 }
 
-export { buildEventOptions };
+const getFormattedPrompt = (userProvidedPrompt: string) =>
+  `GFC Community Member: ${userProvidedPrompt} \n\n GFC Community Software Sparring Partner: `;
```

Comment:
whats a `GFC Community Software Sparring Partner`? and who is it? the bot? a community member? chatgpt system?